exercises/tree-building/examples/success-standard/package.yaml

```diff
@@ -0,0 +1,21 @@
+name:                tree-building
+version:             1.0.0.0
```

Comment:
since there is no canonical-data.json, by what's written in https://github.com/exercism/haskell/blob/master/README.md#exercise-versioning please use `0.1.0.s` instead of `1.0.0.s` version numbers

Reply:
Okay, cool. I wasn't sure of the protocol.